### ISSUE TITLE
Modulation vector for SRN Cars

### ISSUE THREAD

**Issue by tackgeun** [deepmind/functa#12]
Hi, thank you for sharing the code of the great work.

Could you open the modulation vectors for SRN cars?
It would be helpful to examine the implementation of NeRF functionality and future works.

Thank you in advance.

**Comment by hyunjik11** [comment]
Hi Tackgeun, I'm afraid it'll be difficult to share the modulation vectors for SRN cars. I think you may want to implement the dataset generation code yourself by running `experiment_meta_learning.py` on this dataset and then adapting `modulation_dataset_writer.py`. I'd be happy to answer any difficulties you run into while doing this.

**Comment by tackgeun** [comment]
I'll follow your instruction to reproduce the results.
Thank you for your answer.

**Comment by tackgeun** [comment]
I've been trying to reproduce the result of ModulatedSIREN on SRN Cars.
But, the hyper-parameters written in the appendix confuse me a little bit.

Could you inform the hyper-parameter for Latent Modulated SIREN for SRN Cars with FID 80.3?
The publicly-available Latent Modulated SIREN on SRN Cars has 128 modulation dimensions, which is inconsistent with the below two descriptions in the paper.

1. What are the width, depth, and modulation dimensions of Latent modulated SIREN for SRN Cars? And what is the number of views and the number of pixels per view?
>  The optimal hyperparameters for the 512-dimensional modulation dataset is: 32 points per ray, 16 views and 512 pixels per view.

2. Is this FID score evaluated from DDPM with Residual MLP following the description in the appendix?
> For DDPM on 64-dimensional modulations of SRN Cars, we use a batch size of 256 with width 512, 4 blocks and 0. dropout, training for 1e5 iterations

Thank you in advance.

**Comment by hyunjik11** [comment]
1. The width and depth of LatentModulatedSiren for SRN Cars is 512 and 15 respectively. Apologies for missing this out in the paper. The number of views (16) is the number of views per scene that are randomly subsampled (out of 50 views) for each training iteration of meta-learning. Of these 16 views, 512 pixels are subsampled per view.
As stated in the paper, for the modulation dataset creation we have less memory constraints than for the meta-learning, so we use 32 views with 2048 pixels per view.

2. Yes, that is correct. Bear in mind that we use the test set (instead of the training set) for computing the FID score, since the test set contains all possible views whereas the training set only contains randomly subsampled views.

**Comment by tackgeun** [comment]
Thank you for answering the question. I will follow your explanations on experiments.
